### PR TITLE
Language support

### PR DESCRIPTION
**Pull request by satorinic**
### PT-link: https://www.pivotaltracker.com/story/show/150807224 ##
- Add the language support gem i18n.
- Add english text for the menu page/order page introduction.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -1,33 +1,11 @@
...
-# en:
-#   'true': 'foo'
-#
-# To learn more, please read the Rails Internationalization guide
-# available at http://guides.rubyonrails.org/i18n.html.
-
 en:
-  hello: "Hello world"
+  main_title_order_form: "MENU"
+  intro_desc_order_form: "<p class='center'>Our menu is 100% plant-based and vegan. We’re committed to always use
```
Comment (tochman) [inline config/locales/en.yml]: I'm not sure about injecting html into translation files. I would recommend you don't do that.

File: config/locales/en.yml [hunk 2]
```diff
@@ -1,33 +1,11 @@
...
+                          organic ingredients as far as possibly possible. Our packaging is 100%
+                          compostable and our delivery service is made by cargo bike. All our food and
+                          desserts are made without white refined sugar and no added gluten. </p>
+                          <p class='center'>Minimum amount per dish to order - 10 servings. </p>"
+
```
Comment (tochman) [inline config/locales/en.yml]: You can easily split this into two or more strings. Each for one paragraph

File: app/views/home/index.html.erb [hunk 3]
```diff
@@ -1,9 +1,5 @@
-<h1>MENU</h1>
-<p class="center">Our menu is <strong>100% plant-based</strong> and <strong>vegan</strong>.
-  We’re committed to always use organic ingredients as far as possibly possible. Our packaging is 100% compostable and
-  our delivery service is made by cargo bike. All our food and desserts are made without white refined sugar and no
-  added gluten.</p>
-<br>
+<h1><%= I18n.t('main_title_order_form').html_safe %></h1>
+<p class="center"><%= I18n.t('intro_desc_order_form').html_safe %> </p>
```
Comment (tochman) [inline app/views/home/index.html.erb]: Try the shorter `t('intro_desc_order_form')` and by not injecting html into the translation files, you can omit `html_safe`

File: config/application.rb [hunk 4]
```diff
@@ -24,6 +24,9 @@ class Application < Rails::Application
       generate.controller_specs false
     end
 
+    config.i18n.available_locales = :en, :sv
```
Comment (j0sv) [inline config/application.rb]: English is the only language available. Why is 'sv' included?

Comment (satorinic) [reply config/application.rb]: It is not necessary. Just to show that it is possible as a available language. It doesn't effect anything. 
We can remove it.

**Review by tochman — CHANGES_REQUESTED** [review]

File: config/application.rb (hunk 4 above)
Comment (satorinic) [reply config/application.rb]: Removed

**Review by tochman — APPROVED** [review]

**Pull request merged**